spec/models/spree/order_spec.rb

```diff
@@ -162,6 +162,11 @@
 
     describe '#vendor_ids' do
       it { expect(order.vendor_ids).to match_array([vendor.id, vendor_2.id]) }
+
+      context 'when a line item does not belong to any vendor' do
+        before { order.vendor_line_items(vendor2).update_all(vendor_id: nil) }
```

Comment:
```suggestion
        before { order.vendor_line_items(vendor_2).update_all(vendor_id: nil) }
```

typo 😊

Reply:
thank you! :) squashed: https://github.com/spree-contrib/spree_multi_vendor/pull/143/commits/d9b8a56d754328ae9acec061d21f7725ae35ecf0

Reply:
d9b8a56 still CI fails... 🤔  
tried: https://github.com/spree-contrib/spree_multi_vendor/pull/143/commits/628c15e4b3506af004b85595620df54d2a8da220